robottelo/cli/factory.py

```diff
@@ -1784,6 +1783,8 @@ def setup_org_for_a_custom_repo(options=None):
                                     one if empty)
         activationkey-id (optional) - ID of activation key (or create a new one
                                     if empty)
+    :return: A dictionary representing all entities ids used in our setup
+        procedure (like content view or activation key id)
 
```

Comment:
To make this more clear, may be
`A dictionary with the entity ids of Activation key, Content view, LifeCycle Environment, Organization, Product and Repository`